deployments/kubectl/deployment.yaml

```diff
@@ -44,5 +44,25 @@ spec:
           failureThreshold: 10
           timeoutSeconds: 5
           periodSeconds: 30
+      initContainers:
+        - name: fga-store-create
+          image: openfga/cli:v0.2.4
+          command:
+          - /fga
+          - store
+          - create
+          - --api-url
+          - http://10.152.183.205:8080
+          - --api-token
+          - "42"
+          - --name
+          - admin-ui
+          volumeMounts:
+          - name: workdir
+            mountPath: "/fga-store-info"
+      dnsPolicy: Default
+      volumes:
+      - name: workdir
+        emptyDir: {}
```

Comment:
will we still need this or we go full manual with portforwarding, `fga store create` and `kubect edit cm`?

Reply:
removed

Reply:
removed it